docs/releases/1.10.txt

```diff
@@ -600,6 +600,17 @@ You can check if your database has any of the removed hashers like this::
     # Unsalted MD5 passwords might not have an 'md5$$' prefix:
     User.objects.filter(password__length=32)
 
+``TransactionTestCase`` serialized data loading
+-----------------------------------------------
+
+Initial data migrations are now loaded in ``TransactionTestCase`` at the end of
+the test, after the database flush.
+In older versions, these data were loaded at the beginning of the test, but
+this behavior was preventing ``--keepdb`` option to work properly (the
+database was empty at the end of the whole test suite).
+It shouldn't have any impact on your test suite if you have not customized
```

Comment:
I have also tried in last PR update to load the data at the begin AND at the end, after the flush. Tests are passing.

Advantages:

- Old behaviour (loading at the begin) is respected (no backward incompatibility),
- New behaviour (loading at the end) is added,
- Mixing transactions with or without `serialized_rollback` will still work,
- People that want to make it work with `--keepdb` should use always `serialized_rollback` option to `True`.

Inconvenients:

- Each `TransactionTestCase` will be slower than before, as data are loaded twice,
- Surprised that Django is not complaining about loading twice the same data (silent fail ?).